tsconfig.json

```diff
@@ -27,6 +27,7 @@
     "enableI18nLegacyMessageIdFormat": false,
     "strictInjectionParameters": true,
     "strictInputAccessModifiers": true,
-    "strictTemplates": true
+    "strictTemplates": true,
+    "enableIvy": false
```

Comment:
Why did you do that?

Reply:
Angular show me error, when i use pipes. This is fix.